program/src/main/System.vala

```diff
@@ -17,11 +20,18 @@ public class System
     public static Gtk.ProgressBar progress_bar; //progress bar for user to view how many questions are graded out of total number of questions
 
     public static Gtk.Grid marksGrid; //the grid that shows all of the current questions marks
+    public static Gtk.ActionBar page_nums; //Entry box for the value of examPagesPerTest on setup
     public static bool isBindingNewMarkHotkey = false;
     public static MarkViewHotkeyButton markHotkeyButton;
-    
-    private System(){}
 
+    public static Gtk.RadioButton name_radio; //Radio button with name, needs to be global so other radio buttons can be added to the set of radio buttons
+
```

Comment:
Remove these extra spaces

Reply:
I removed the excessive lines between most of the fields. I still kept some of them in groups based on function/feature so that it's easier to parse at a glance. Let me know if this is not preferred.